src/generators/randgraphs.jl

```diff
@@ -31,6 +31,20 @@ function DiGraph(nv::Integer, ne::Integer; seed::Int = -1)
     return g
 end
 
+# taken from http://stackoverflow.com/questions/23561551/a-efficient-binomial-random-number-generator-code-in-java
+function _getBinomial(n::Integer, p::Real, seed::Integer=-1)
```

Comment:
Can we change the name to `randbn` less JAVA please 😄. I don't think there is a need for `_` prefix, we aren't exporting the function so people will know not to depend on that api.